Minio/MinioClient.cs

```diff
@@ -411,10 +411,7 @@ private async Task<ResponseResult> ExecuteTaskCoreAsync(
     {
         var startTime = DateTime.Now;
         // Logs full url when HTTPtracing is enabled.
-        if (trace)
-        {
-            var fullUrl = requestMessageBuilder.RequestUri;
-        }
+        if (trace) _ = requestMessageBuilder.RequestUri;
```

Comment:
Why not remove this all together? What's the point of doing this?

Reply:
ok. Done!